src/docs/asciidoc/en/rlsnotes/rlsnotes40/_dml-40.adoc

```diff
@@ -35,6 +35,17 @@ The feature is defined in (SQL:2011): 7.6 <table reference> (Feature T491).
 .Examples
 [source]
 ----
+/* select customers with they last order date and number */
+select c.name, ox.order_date as last_order, ox.number
+from customer c
+  left join LATERAL
+  (
+  select first 1 o.order_date, o.number
+  from orders o
+  where o.id_customer=o.id
```

Comment:
Not c.id instead of o.id?
where o.id_customer = c.id

Reply:
You're right, I had overlooked that. Fix is underway.

Reply:
Fixed in df16def17ad859094380850b50109ed540daab38, thanks for reporting.